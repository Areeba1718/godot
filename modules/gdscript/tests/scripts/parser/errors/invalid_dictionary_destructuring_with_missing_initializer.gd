func test():
    var {foo = a, bar = b}
