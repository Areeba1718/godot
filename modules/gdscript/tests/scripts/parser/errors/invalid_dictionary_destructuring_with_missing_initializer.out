GDTEST_PARSER_ERROR
Expected assignment ("=") after destructuring.
